paint/example/src/lib.rs

```diff
@@ -482,6 +482,15 @@ decl_module! {
 			Ok(())
 		}
 
+		// TODO: this stuff will move to a new test.
+		#[weight = sr_primitives::weights::FunctionOf(
+			|args: (&u32, &T::AccountId, &BalanceOf<T>)| (args.0 + 100) as Weight,
```

Comment:
Is it possible to have type inferring to avoid unnecessary code?

```suggestion
			|(val, _, _)| val + 100,
```

Reply:
Sadly, as far s I tried, no